python/foxglove-sdk-examples/live-visualization/pyproject.toml

```diff
@@ -5,6 +5,7 @@ description = "Example of live visualization in Foxglove"
 authors = [{ name = "Foxglove", email = "support@foxglove.dev" }]
 readme = "README.md"
 requires-python = ">=3.9"
+dependencies = ["foxglove-sdk>=0.4.0", "numpy>=2.0"]
```

Comment:
Is there a way we could override this in CI so that the examples are tested (or at least type checked) against the main library?

Reply:
I don't think this is well-supported in poetry, so I updated the script to install the local version when prompted. 

https://github.com/foxglove/foxglove-sdk/actions/runs/14111136584/job/39530020221#step:13:39